src/quicksort.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+# -*- coding: utf -8 -*-
+
+from __future__ import unicode_literals
+from random import choice
+
+
+def _sort_relative_to_pivot(a_list, pivot_pos):
+    """
+    Sort items in the list relative to the pivot.
+    Return position of the pivot after the sort completed.
+    """
+    pivot = a_list[pivot_pos]
+    i = 0
```

Comment:
Slight criticism here: it would be more readable to set more meaningful variable names here that `i` and `x`, so anyone reading your code might get a hint of the purpose of those variables.